packages/components/checkbox/src/composables/use-checkbox-event.ts

```diff
@@ -52,11 +52,19 @@ export const useCheckboxEvent = (
     if (isLimitExceeded.value) return
 
     if (!hasOwnLabel.value && !isDisabled.value && isLabeledByFormItem.value) {
-      model.value = getLabeledValue(
-        [false, props.falseLabel].includes(model.value)
+      // fix: https://github.com/element-plus/element-plus/issues/9981
+      const eventTargets: Array<EventTarget> =
+        e.composedPath && e.composedPath()
```

Comment:
Is the judgment here necessary? How about simplifying it like this:
```suggestion
        e.composedPath()
```